src/modules/CRUD/components/Details/PropertyItem/Edit/Edit.js

```diff
@@ -69,8 +69,13 @@ const Edit = ({
       return sanitizeValue(schema, displayValue);
     }
     const { properties = {} } = feature[id] || {};
-    const propertiesListed = Object.values(properties).reduce(
-      (list, item) => ({ ...list, ...item }),
+    const propertiesListed = Object.entries(properties).reduce(
```

Comment:
Can you add a comment to explain what this code is actually doing?

Reply:
Done 👍